### ISSUE TITLE
Model scaffold doesn't import all of the helper functions for model definition

### ISSUE THREAD

**Issue by slyons** [loco-rs/loco#443]
**Description**

Using the following command:

```
❯ cargo loco generate scaffold todos title:string! done:bool! user:references
```

I get the following error:

```
    Finished dev [unoptimized + debuginfo] target(s) in 0.67s
     Running `target/debug/loco_leptos_todomvc-cli generate scaffold todos 'title:string'\!'' 'done:bool'\!'' 'user:references'`
added: "migration/src/m20240214_213812_todos.rs"
injected: "migration/src/lib.rs"
injected: "migration/src/lib.rs"
added: "tests/models/todos.rs"
injected: "tests/models/mod.rs"
   Compiling migration v0.1.0 (/opt/dev/projects/loco_leptos_todomvc/migration)
error[E0423]: expected function, found builtin type `bool`
  --> migration/src/m20240214_213812_todos.rs:14:26
   |
14 |                     .col(bool(Todos::Done))
   |                          ^^^^ not a function
   |
help: consider importing this function instead
   |
1  + use loco_rs::schema::bool;
   |
```

The generated file doesn't have that `use` statement and the scaffolding migration process fails

```rust
use sea_orm_migration::{prelude::*, schema::*};

#[derive(DeriveMigrationName)]
pub struct Migration;

#[async_trait::async_trait]
impl MigrationTrait for Migration {
    async fn up(&self, manager: &SchemaManager) -> Result<(), DbErr> {
```

**To Reproduce**

Run the above command in your Loco project

**Expected Behavior**

A new generated model and controller scaffolding

**Comment by clefelhocz2** [comment]
I reviewed this one and it looks like it was reproduced under 0.3.1.   Due to #457 this issue has been fixed in 0.3.2.  @slyons if you have a minute to confirm with the new crate it would be useful.

In the process though I noticed that #457 incorrectly updated the documentation to say to use boolean, boolean! rather than bool, bool!, so we need to fix that.

**Comment by kaplanelad** [comment]
closing this issue. 
if you stil have a problem fell free to repoen or create a new one
